src/app/cli/src/init/client.ml

```diff
@@ -561,21 +562,28 @@ let user_command (body_args : User_command_payload.Body.t Command.Param.t)
          let fee =
            Option.value ~default:Cli_lib.Default.transaction_fee fee_opt
          in
-         let memo =
-           Option.value_map memo_opt ~default:User_command_memo.empty
-             ~f:User_command_memo.create_from_string_exn
-         in
-         let command =
-           Coda_commands.setup_user_command ~fee ~nonce ~memo ~sender_kp body
-         in
-         dispatch_with_message Daemon_rpcs.Send_user_command.rpc command port
-           ~success:(fun receipt_chain_hash ->
-             sprintf "Dispatched %s with ID %s\nReceipt chain hash is now %s\n"
-               label
-               (User_command.to_base58_check command)
-               (Receipt.Chain_hash.to_string receipt_chain_hash) )
-           ~error:(fun e -> sprintf "%s: %s" error (Error.to_string_hum e))
-           ~join_error:Or_error.join ))
+         if Currency.Fee.( < ) fee User_command.minimum_fee then (
+           printf "Fee %d is less than the minimum, %d\n%!"
+             (Currency.Fee.to_int fee)
+             (Currency.Fee.to_int User_command.minimum_fee) ;
+           Deferred.unit )
```

Comment:
Maybe this should exit with a non-zero integer? The transaction did fail to send after all.